apps/barista-design-system/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -68,6 +68,7 @@ export class BaIconColorWheel {
 
   constructor(private _sanitizer: DomSanitizer) {
     const groupedBlobs = Object.keys(DtColors)
+      .filter(key => key !== 'FLAT_WHITE') // no flat white for the color wheel
```

Comment:
To be honest, I would remove the colour from the scss file :D

Reply:
And add a comment to the scss file that every added colour will be displayed in the colour wheel otherwise we will forget in the future.

Reply:
I agree with @lukasholzer here that this is an unnecessary risk. It was fun while it lasted :D Let's cleanup the color, remove the filter and add a comment in the colors to check the color wheel once something changes.

Reply:
Need to keep it in, because removing the color would be a breaking change.